source/developers/references/developer_guide/process/landing-your-work.rst

```diff
@@ -0,0 +1,135 @@
+*****************
+Landing Your Work
+*****************
+
+Once you're ready to submit your code changes in a pull request, check the following
+requirements to be sure that your pull request is ready to be reviewed:
+
+---------------------
+Describe your changes
+---------------------
+
+Prepare a detailed pull request description. Most repositories have a template
+to be followed when you open the PR in the GitHub UI. Please fill out that template
+with as much detail as possible, including links to any roadmap tickets or
+forum discussions as well as screenshots or screencasts (not everyone reviewing
+your changes is a developer).
+
+The code should be clear and understandable. Comments in code, detailed
+docstrings, and good variable naming conventions are expected. See the
+:doc:`../style_guides/index` for more details.
+
+Commit messages should conform to `OEP-51: Conventional Commits`_.
+This style categorizes commits to make them easier to understand.
+
+----------------
+Limit your scope
+----------------
+
+The pull request should be as small as possible. Each pull request should
+encompass only one idea: one bug fix, one feature, etc. Multiple features
+(or multiple bug fixes) should not be bundled into one pull request. A
+handful of small pull requests is much better than one large pull request.
+
+Structure your pull request into logical commits. “Fixup” commits
+should be squashed together. The best pull requests contain only a
+single, logical change - which means only a single, logical
+commit. You can squash your own commits, or the person merging the
+pull request may choose to squash them.
+They will do so by using the Squash and merge button of
+the GitHub interface to preserve the logical commits in the pull
+request, for forensic purposes when trying to diagnose a regression
+or understand a bug.
+
+The pull request should integrate with existing infrastructure as much as
+possible, rather than reinventing the wheel. In a project as large as ours,
+there are many foundational components that might be hard to find.
+It is important not to duplicate functionality, even if small, that already
+exists. Posting in the `forums`_ is an easy way to find out if functionality
+you're looking for already exists.
+
+---------
+Licensing
+---------
+
+All code in the pull request must be compatible with the Open edX project's AGPL
+license.  This means that the author of the pull request must sign a
+`contributor's agreement`_, and all libraries included or
```

Comment:
```suggestion
`contributor's agreement`_ (or be covered by their employer's agreement), and all libraries included or
```
Adding this because I know in the past we've had some slightly confused engineers at other companies submit a CLA when they don't need to.